tests/ir/.gitignore

```diff
@@ -0,0 +1 @@
+/Output
```

Comment:
I can't judge this, because I don't know how much of the test result depends on libd/libd-llvm/somethingelse? 
It may be nicer to keep all tests together in the repo, so then perhaps /tests/libd-llvm ?